DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/AvatarOperations.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtention.Preferences.WebApi.Constants;
+using GitHubExtention.Preferences.WebApi.Extensions;
+using GitHubExtention.Preferences.WebApi.Models;
+using GitHubExtention.Preferences.WebApi.Queries;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public static class AvatarOperations
+    {
+        public static async Task<string> GetNewAvatarUrl(this IAzureContainerQuery container, string url)
```

Comment:
Method with a Get prefix but inside the body it;s call UpdateBlob? Can you clarify this....